turbo.json

```diff
@@ -1,5 +1,6 @@
 {
-  "pipeline": {
+  "globalEnv": ["HOST_JOBS_DIR", "JOBS_DIR"],
```

Comment:
https://turbo.build/repo/docs/crafting-your-repository/using-environment-variables#adding-environment-variables-to-task-hashes

Reply:
Based on https://turbo.build/repo/docs/crafting-your-repository/using-environment-variables#environment-modes. 
An alternative is to [switch turbo to "loose mode"](https://turbo.build/repo/docs/crafting-your-repository/using-environment-variables#loose-mode), so that all environment variables are copied to the processes.

Reply:
Your solution seems good to me

Reply:
Question is: is there another environment variable that tests (or build) rely on?

Reply:
I grepped for `process.env`. The only thing that jumped out at me is `process.env.MOCHA_WORKER_ID` to run parallel tests, but that might be ok because it's being set by mocha which is already inside turbo? There is also `TSX_TSCONFIG_PATH` used in `apps/prairielearn/.mocharc.cjs`, but I think this is similarly ok because mocha is setting it explicitly.

Other environment variables that we use are `NODE_ENV`, `CONFIG_LOAD_FROM_AWS`, `DEBUG`, `PROFILE_SYNC`, `AWS_ENDPOINT`, and `PL_KEEP_TEST_DB`, but I don't think any of these are needed for CI?

Reply:
I have changed the file to passthrough all environment variables to test/test:dist. This way we can have test/test:dist work with a similar set of environment variables as production without relying on them being explicitly listed in turbo.json.

Reply:
Nice, that sounds super easy. I like it!